students/11th/kungyulee/posts/models.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime 
+
+from django.db import models
+
+from accounts.models import User
+
+class Post(models.Model):
+    author    = models.ForeignKey(User, on_delete = models.CASCADE)
+    contents  = models.TextField()
+    image_url = models.URLField(max_length = 200, blank = False)
+    date = models.DateTimeField(auto_now_add=True)
```

Comment:
코드 간격 맞춰주세요
생성 날짜라는 속성은 통상 `created_at`을 많이 사용합니다.
또한, 포스팅은 수정도 할 수 있기 때문에 `updated_at`이라는 수정된 날짜 기록하는 속성도 추가해보세요 :)
```suggestion
    author    = models.ForeignKey(User, on_delete = models.CASCADE)
    contents  = models.TextField()
    image_url = models.URLField(max_length = 200, blank = False)
    created_at = models.DateTimeField(auto_now_add=True)
```